openshift/osd-network-verifier
# [Feature] Add script to clean up golden AMIs in AWS regions where we're at our quota for public images

## What does this PR do? / Related Issues / Jira

Adds a Go script to clean up our Golden AMIs in regions where we've hit our quota.

## Checklist

## How to test this PR locally / Special Instructions

First, make sure you have AWS credentials for the target account. Then cd into `cleangoldenami`, run `go build`, then run the executable (preferably with `--dry-run` and `--verbose`)

## Logs 

```
Region af-south-1 is under quota. (Images: 13, Quota: 20)
Region ap-east-1 is under quota. (Images: 14, Quota: 20)
Region ap-northeast-1 is under quota. (Images: 14, Quota: 20)
Region ap-northeast-2 is under quota. (Images: 13, Quota: 20)
Region ap-northeast-3 is under quota. (Images: 14, Quota: 20)
Region ap-south-1 is under quota. (Images: 14, Quota: 20)
Region ap-south-2 is under quota. (Images: 8, Quota: 20)
Region ap-southeast-1 is under quota. (Images: 15, Quota: 20)
Region ap-southeast-2 is under quota. (Images: 13, Quota: 20)
Region ap-southeast-3 is under quota. (Images: 13, Quota: 20)
Region ap-southeast-4 is under quota. (Images: 8, Quota: 20)
Region ca-central-1 is under quota. (Images: 13, Quota: 20)
Region eu-central-1 is at quota (10) - would delete ami-08eca5bab97189074
Region eu-central-2 is under quota. (Images: 8, Quota: 20)
Region eu-north-1 is under quota. (Images: 9, Quota: 20)
Region eu-south-1 is under quota. (Images: 12, Quota: 20)
Region eu-south-2 is under quota. (Images: 7, Quota: 20)
Region eu-west-1 is under quota. (Images: 13, Quota: 20)
Region eu-west-2 is under quota. (Images: 14, Quota: 20)
Region eu-west-3 is under quota. (Images: 14, Quota: 20)
Region me-central-1 is under quota. (Images: 6, Quota: 20)
Region me-south-1 is under quota. (Images: 12, Quota: 20)
Region sa-east-1 is under quota. (Images: 13, Quota: 20)
Region us-east-1 is under quota. (Images: 11, Quota: 20)
Region us-east-2 is under quota. (Images: 15, Quota: 20)
Region us-west-1 is under quota. (Images: 12, Quota: 20)
Region us-west-2 is under quota. (Images: 14, Quota: 20)
Done!
```
